### PR TITLE
main page content provision

### PR DESCRIPTION
**Pull request by RamziKaroui**
Signed-off-by: Ramzi KAROUI <ramzi.karoui@adlinktech.com>

main page content creation

### CODE REVIEW

File: content/pages/index.rst [hunk 1]
```diff
@@ -10,79 +10,57 @@ Home
...
-**feature 1**
-Description for feature 1
+Fast & Dependable
+-----------------
 
-**feature 2**
-Description for feature 2
+Cyclone DDS has an extremely low latency and high throughput, especially in rush environments and noisy networks where error transmission rates are relatively high.​
+Performance variability and jitters are minimum which make it suitable for real-time and mission critical applications.​
+Cyclone also has a low footprint making it suitable for both enterprise and embedded systems.​
```
Comment (k0ekk0ek) [inline content/pages/index.rst:40]: Please use *Cyclone DDS* consistently. I know everybody knows we mean Cyclone DDS when we say Cyclone but still. The official name is Cyclone DDS. (Should of course have been *Splice*, but still).

File: content/pages/index.rst [hunk 2]
```diff
@@ -10,79 +10,57 @@ Home
...
-High performant and robust open source OMG DDS implementation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Brief summary of what makes Eclipse Cyclone DDS unique.
+.. class:: col-10  mx-auto col-md-5 order-md-2 text-center  pr-md-5
 
-.. class:: ato-button
+Building Portable & Interoperable Distributed Datacentric Systems
+-----------------------------------------------------------------
+Cyclone DDS is a high performing, OMG-DDS standard based data sharing technology which allows system designers to create digital twines of their systems’ entities to share their states, events, data-streams and messages on the network in real-time and fault-tolerant way
```
Comment (k0ekk0ek) [inline content/pages/index.rst:25]: twines >> twins

**Review by k0ekk0ek — CHANGES_REQUESTED** [review]

**Comment by k0ekk0ek** [comment]
Thanks @RamziKaroui! Thanks for providing some content :+1: Couple of minor things, other than that, this seems absolutely fine.

**Comment by thijsmie** [comment]
I have a couple of comments on the css styling here: the original content was carefully constructed to get the `container > row > col-*` structure. This is not quite right in this commit, meaning you will see a horizontal scrollbar appear because the content is not correctly fixed-width to the viewport.
Next, the change of the `col-6 col-md-2 pb-2` to `col-md-2` messes with the responsive setup of the website. The point of these  is that the first class `col-*` select the rendering from the smallest viewports and then the `col-md-*` variants from medium viewports and up, as explained [here](https://getbootstrap.com/docs/5.0/layout/grid/). I recommend you view the site with the [chrome mobile device simulator](https://developer.chrome.com/docs/devtools/device-mode/) in a bunch of sizings to get a feel for this.

**Comment by k0ekk0ek** [comment]
@thijsmie, it looks decent if I run it in the responsive mode in Firefox. Is the css ok now or do we still require changes?

**Comment by thijsmie** [comment]
My requested changes were made so the CSS is okay now.

**Pull request merged**